test/run_e2e_test.py

```diff
@@ -263,4 +268,5 @@ def main(args=None) -> int:
 if __name__ == '__main__':
     setup_logger()
     test_output = main()
+    logging.debug("exiting with test_output=%s", test_output)
```

Comment:
Why not use `.format()`?

Reply:
Simply using the logging API: https://docs.python.org/3/library/logging.html#logging.Logger.debug